packages/ui/src/theme/tokens/components/fieldControl.js

```diff
@@ -2,7 +2,7 @@ module.exports = {
   borderStyle: { value: 'solid' },
   borderColor: { value: '{colors.border.primary.value}' },
   borderWidth: { value: '{borderWidths.small.value}' },
-  borderRadius: { value: '{radii.medium.value}' },
+  borderRadius: { value: '{radii.small.value}' },
```

Comment:
Does this change affect other components? Any reason why we want to make the radii smaller?

Reply:
I updated the tokens. The `medium` value goes into `small` now